ports/esp32/boards/GENERIC_C3/deploy.md

```diff
@@ -0,0 +1,14 @@
+Program your board using the esptool.py program, found [here](https://github.com/espressif/esptool).
```

Comment:
It would be good to move this file to `ports/esp32/boards/deploy_c3.md`, to match `deploy_s2.md`.  And then reference it from both these C3 boards.

Reply:
it shall be done